README.md

```diff
@@ -55,6 +58,18 @@ See [`syntax.md`](syntax.md) for the assembly language syntax.
 
 *Warning:* The validator is incomplete.
 
+### Optimizer
+
+*Warning:* The optimizer is still under development and its library interface is
+not yet published.
+
+Currently supported optimizations:
+* [Strip debug info](source/opt/strip_debug_info_pass.h)
+* [Freeze spec constant](source/opt/freeze_spec_constant_value_pass.h)
+* [Fold `OpSpecConstantOp` and `OpSpecConstantComposite`](source/opt/fold_spec_constant_op_and_composite_pass.h)
+* [Unify constants](source/opt/unify_const_pass.h)
+* [Eliminate dead constant](source/opt/eliminate_dead_constant_pass.h)
+
```

Comment:
Suggest that for the latest list, run 'spirv-opt --help'
Since this list will get out of date pretty easily.